wgpu-hal/src/dx12/command.rs

```diff
@@ -319,15 +321,29 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
                     // Only one barrier if it affects the whole image.
                     self.temp.barriers.push(raw);
                 } else {
-                    // Generate barrier for each layer/level combination.
+                    let has_stencil = FormatAspects::from(barrier.texture.format)
+                        .contains(FormatAspects::STENCIL);
+                    let planes = if has_stencil {
+                        match barrier.range.aspect {
+                            wgt::TextureAspect::All => 0..2,
+                            wgt::TextureAspect::StencilOnly => 1..2,
```

Comment:
what if it's a stencil-only format? would it still be 1..2?

Reply:
huh, yeah that should be 0..1 as well I suppose! Fixed!